zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -225,26 +225,13 @@ fn setup_resolve_common_scalar_after_dispute<T: Config>(
 
 benchmarks! {
     admin_destroy_disputed_market{
-        // a = total accounts
-        // An higher number increases the benchmark runtime significantly, while increasing the
-        // error due to a lower number of repetitions (the data points that are used to approximate
-        // the weight function weight(a) are less precise).
-        // The required weight per account is a linear function of degree 1, i.e.
-        // fn(a) = ba^1 + ca^0. The first few data points of the curve are already approximating
-        // the function and its constant gradient fairly well, provided that these few data points
-        // are very precise (many repetitions). If the gradient is well estimated, any value can be
-        // derived, up to infinity, assuming that at no point the function fn(a) is non-linear.
-        let a in 0..10;
-        // b = num. accounts with assets
-        // Unfortunately frame-benchmarking does not allow to b = b.min(a) here
-        let b in 0..10;
-        // c = num. asset types
-        let c in (T::MinCategories::get().into())..T::MaxCategories::get().into();
-        // Complexity: c*a + c*b ∈ O(a)
-        let c_u16 = c.saturated_into();
-        let (caller, market_id) = setup_resolve_common_categorical::<T>(a, b, c_u16)?;
-
-        for i in 0..c.min(T::MaxDisputes::get()) {
+        let total_accounts = 1000;
+        let accounts_with_assets = 750;
+        let categories = T::MaxCategories::get();
+        let (caller, market_id) = setup_resolve_common_categorical::<T>(total_accounts, accounts_with_assets, categories)?;
```

Comment:
We don't need any outcome token holding account now, so we can remove the overhead of creating them in the benchmarks (which get executed thousands of times):
```suggestion
        let categories = T::MaxCategories::get();
        let (caller, market_id) = setup_resolve_common_categorical::<T>(0, 0, categories)?;
```

Reply:
Or just use the following.

```rust
let categories: u16 = T::MaxCategories::get().saturated_into();
let (caller, market_id) = create_close_and_report_market::<T>(
    MarketCreation::Permissionless,
    MarketType::Categorical(categories),
    OutcomeReport::Categorical(categories.saturating_sub(1)),
)?;
```